assets/src/containers/Course.js

```diff
@@ -65,6 +68,8 @@ function Course (props) {
                         {...props}
                         courseInfo={courseInfo}
                         courseId={courseId}
+                        enrollmentType={enrollmentType}
+                        isAdmin={user.admin}
```

Comment:
Question: do we need to know whether they are specifically an instructor or admin within `IndexPage` or `SideDrawer`? Could we just consolidate into an `instructorOrAdmin` or `teacherOrAdmin` boolean prop? This way, you may be able to just call your `isTeacherOrAdmin` utility function here in `Course.js` and pass the result where necessary.

Reply:
Currently there is no need within those contexts do distinguish which role is granting them the functionality so your suggestion would be fine.  It seems like there could be functionality down the road that does distinguish between the two, but who knows.  I think it's fine either way now, but I don't feel particularly motivated to change it.  If it seems important I can go ahead and do it.

Reply:
Seems like it could simplify the code, and the number of places you call your utility function, but it's not a high priority. One potential argument in favor of doing it this way would be because really it should be `enrollmentTypes` (see other comments), so maybe that's another level of input complexity that you'd avoid by moving the processing up in the application.